### PR TITLE
Jpuerto psc/schema examples

### PR DESCRIPTION
**Pull request by jpuerto-psc**
This PR adds examples to the schemas for fields that have pattern constraints and do not currently have examples listed.

### CODE REVIEW

File: docs/scatacseq/v1.yaml [hunk 1]
```diff
@@ -313,23 +314,25 @@ fields:
     forbid_na: true
 - name: cell_barcode_offset
   constraints:
+    required: false
```
Comment (jpuerto-psc) [inline docs/scatacseq/v1.yaml:317]: Needs to be true

File: docs/scatacseq/v1.yaml [hunk 2]
```diff
@@ -313,23 +314,25 @@ fields:
...
     offsets). First barcode at position 0, then 38, then 76. (Does not apply to sciATACseq,
     SNARE-seq and BulkATAC.)
+  example: 0,0,38,76
   custom_constraints:
     sequence_limit: 3
     forbid_na: true
   type: string
 - name: cell_barcode_size
   constraints:
+    required: false
```
Comment (jpuerto-psc) [inline docs/scatacseq/v1.yaml:330]: Needs to be true

File: docs/scrnaseq/v2.yaml [hunk 3]
```diff
@@ -302,19 +304,11 @@ fields:
   constraints:
     required: false
     pattern: \d+(,\d+)*
+  example: 16,8,8,8
   custom_constraints:
     sequence_limit: 3
     forbid_na: true
   type: string
-- name: expected_cell_count
```
Comment (jpuerto-psc) [inline docs/scrnaseq/v2.yaml:309]: This should be added back, not sure why it got deleted

File: src/ingest_validation_tools/table-schemas/assays/scrnaseq-v2.yaml [hunk 4]
```diff
@@ -79,12 +80,13 @@ fields:
   constraints:
     required: FALSE
     pattern: '\d+(,\d+)*'
+  example: 0,0,38,76
 - name: cell_barcode_size
   description: Length of the cell barcode in base pairs
   constraints:
     required: FALSE
     pattern: '\d+(,\d+)*'
-# include: ../includes/fields/expected_cell_count.yaml
```
Comment (jpuerto-psc) [inline src/ingest_validation_tools/table-schemas/assays/scrnaseq-v2.yaml:87]: Needs to be added back

File: docs/scatacseq/v1.yaml (hunk 2 above)
Comment (bhonick) [reply docs/scatacseq/v1.yaml:330]: Amended.

File: src/ingest_validation_tools/table-schemas/assays/scrnaseq-v2.yaml (hunk 4 above)
Comment (bhonick) [reply src/ingest_validation_tools/table-schemas/assays/scrnaseq-v2.yaml:87]: Amended.

**Review by bhonick — APPROVED** [review]

**Review by sunset666 — APPROVED** [review]

**Pull request merged**